enterprise_access/apps/api/serializers/subsidy_access_policy.py

```diff
@@ -672,17 +672,24 @@ def get_group_associations(self, obj):
         return list(obj.groups.values_list("enterprise_group_uuid", flat=True))
 
 
-class SubsidyAccessPolicyCanRedeemReasonResponseSerializer(serializers.Serializer):
+class SubsidyAccessPolicyCanRedeemDisplayReasonSerializer(serializers.Serializer):
     """
-    Response serializer used to document the structure of a "reason" a content key is not redeemable, used by the
-    can_redeem endpoint.
+    Response serializer used to document the structure of a "display_reason" highlighting a single reason
+    for policy non-redeemability
     """
     reason = serializers.CharField(
         help_text="reason code (in camel_case) for why the following subsidy access policies are not redeemable."
     )
     user_message = serializers.CharField(
         help_text="Description of why the following subsidy access policies are not redeemable."
     )
```

Comment:
Why not just return a SubsidyAccessPolicyCanRedeemReasonResponseSerializer under `display_reason`?  The same frontend code and backend view code should work, and we'll at least get a clue as to which policy was selected for the display_reason.

Reply:
Updated 👍🏽